.github/workflows/dockershim.yml

```diff
@@ -0,0 +1,123 @@
+name: critest dockershim
+on: [push, pull_request]
+jobs:
+  #
+  # Run CRI tests against dockershim
+  #
+  build-and-critest-dockershim:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04] # Need to fix windows-2019 critest
+                           # It was also disabled in travis ci
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Install Go 1.14
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.14'
+
+      - name: Set Go binary path
+        shell: bash
+        run: |
+          echo "::set-env name=GOPATH::${{ github.workspace }}"
+          echo "::add-path::${{ github.workspace }}/bin"
+
+      - name: Checkout cri-tools for this commit
+        uses: actions/checkout@v2
+        with:
+          path: ${{github.workspace}}/src/github.com/kubernetes-sigs/cri-tools
+
+      - name: Install docker
+        run: |
+          hack/install-docker.sh
+        working-directory: ${{ github.workspace }}/src/github.com/kubernetes-sigs/cri-tools
+
+      - name: Build cri-tools on Linux
+        if: startsWith(matrix.os, 'ubuntu')
+        run: |
+          make
+          sudo PATH=$PATH GOPATH=$GOPATH make install
+        working-directory: ${{ github.workspace }}/src/github.com/kubernetes-sigs/cri-tools
+
+      - name: Build cri-tools on Windows
```

Comment:
let's make a backup of this windows stuff somewhere and remove it from here since it can't be tested yet

Reply:
I think that it might be better leave the Windows steps in place, so then it just needs to be tweaked afterwards to get it to work. Otherwise, you are putting it somewhere else which can be forgotten afterwards. It keeps it also in view to remind that it needs fixing.

Reply:
Open to opinions on it.

Reply:
After discussion with @mikebrow , probably makes sense as it's not working to extract it for now. I have opened PR #634 to handle this.